pkg/apic/servicebuilder.go

```diff
@@ -203,11 +204,17 @@ func (b *serviceBodyBuilder) SetUnstructuredFilename(filename string) ServiceBui
 	b.serviceBody.UnstructuredProps.Filename = filename
 	return b
 }
+
 func (b *serviceBodyBuilder) SetAltRevisionPrefix(revisionPrefix string) ServiceBuilder {
 	b.serviceBody.AltRevisionPrefix = revisionPrefix
 	return b
 }
 
+func (b *serviceBodyBuilder) SetOrganizationName(id string) ServiceBuilder {
+	b.serviceBody.OrganizationName = id
+	return b
+}
+
```

Comment:
Can this be SetTeamName?  In build check if a team name was set (and if the team exists in the cache) otherwise use the default team.  Then the service body you can just use the team name without additional checks?

Reply:
Yeah. I see now that the var in ServiceBody should be called TeamName also, rather than OrganizationName, as it ultimately becomes OwningTeam in the instanceSpec. So, I'll do some renaming, etc.

Reply:
Jason's comment about moving the checks proved to not be possible without breaking changes to other agents caused by cyclic imports. After discussing the problem, we decided to leave it pretty much as-is, except for the renaming.